src/ConfigFactory.Avalonia.Demo/ConfigFactory.Avalonia.Demo.csproj

```diff
@@ -28,4 +28,20 @@
   <ItemGroup>
     <ProjectReference Include="..\ConfigFactory.Avalonia\ConfigFactory.Avalonia.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <AvaloniaResource Update="Assets\Translations.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Translations.resx</DependentUpon>
+    </AvaloniaResource>
+    <AvaloniaResource Update="Assets\Translations.th-TH.resx">
+      <Generator></Generator>
+    </AvaloniaResource>
+  </ItemGroup>
+  <ItemGroup>
+    <AvaloniaResource Update="Assets\Translations.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>Translations.Designer.cs</LastGenOutput>
+    </AvaloniaResource>
```

Comment:
These aren't needed at all and should be removed to avoid confusion (tested with only the top entry and it works fine)

```suggestion
```